routes/helpers/request-validators.js

```diff
@@ -34,6 +34,6 @@ class RequestValidators {
     }
 }
 
-RequestValidators.MAX_NUM_SECTIONS = 50;
+RequestValidators.MAX_NUM_SECTIONS = process.env.MAX_NUM_SECTIONS ?? 50;
```

Comment:
@TaiPhamD unfortunately the project hasn't upgraded to node 14 and can't use `??`

Reply:
fixed it to just if-else statement